<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="choose_method">
          <item>
           <property name="text">
            <string>Hash </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Features</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Both</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_mix1">
          <property name="text">
           <string>open mix 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_mix2">
          <property name="text">
           <string>open mix 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="Mixer_songs">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="percentagemix">
          <property name="text">
           <string>percentage_mix</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="similarity_output">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Similarity Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="similarity_output_table">
          <column>
           <property name="text">
            <string>Song Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Similarity (%)</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="show_results">
          <property name="text">
           <string>Show Results</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_Songs">
    <property name="title">
     <string>Open Song</string>
    </property>
   </widget>
   <addaction name="menuOpen_Songs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
